CHANGELOG

```diff
@@ -234,11 +228,11 @@ Miscellaneous
 
 - Added unit tests for time integrators.
 
-Backward incompatible API changes
----------------------------------
-- In the enum classes MemoryType and MemoryClass, renamed the 'CUDA' to 'DEVICE'
+API changes
+-----------
+- In the enum classes MemoryType and MemoryClass, 'CUDA' was renamed to 'DEVICE'
   which now denotes either 'CUDA' or 'HIP' depending on the build configuration.
-  In the same enum classes, renamed 'CUDA_UVM' to 'MANAGED'.
+  In the same enum classes, 'MANAGED' was renamed to 'CUDA_UVM'.
```

Comment:
This is incorrect now --`CUDA_UVM` was renamed to `MANAGED` -- not the other way around.